Mathlib/Algebra/Order/Ring/Basic.lean

```diff
@@ -282,6 +282,18 @@ lemma Odd.strictMono_pow (hn : Odd n) : StrictMono fun a : R => a ^ n := by
   refine lt_of_add_lt_add_right (a := a + b) ?_
   rwa [add_rotate', ← hbd, add_zero, add_left_comm, ← add_assoc, ← hac, zero_add]
 
+lemma Odd.pow_injective {n : ℕ} (hn : Odd n) : Injective (· ^ n : R → R) :=
+  hn.strictMono_pow.injective
+
+lemma Odd.pow_lt_pow {n : ℕ} (hn : Odd n) {a b : R} : a^n < b^n ↔ a < b :=
+  hn.strictMono_pow.lt_iff_lt
+
+lemma Odd.pow_le_pow {n : ℕ} (hn : Odd n) {a b : R} : a^n ≤ b^n ↔ a ≤ b :=
+  hn.strictMono_pow.le_iff_le
+
+lemma Odd.pow_eq_pow {n : ℕ} (hn : Odd n) {a b : R} : a^n = b^n ↔ a = b :=
```

Comment:
```suggestion
lemma Odd.pow_inj{n : ℕ} (hn : Odd n) {a b : R} : a^n = b^n ↔ a = b :=
```
See the [naming convention](https://leanprover-community.github.io/contribute/naming.html#injectivity).